web-animations-2/Overview.bs

```diff
@@ -226,18 +226,12 @@ Append:
 
 > Animation effects associated with a [=progress-based timeline=] require their
 > timing properties to be converted to proportions. The procedure for converting
-> a <dfn lt="time-based animation to a proportional animation"></dfn> is as
+> a <dfn lt="time-based animation to a proportional animation">time-based
+> animation to a proportional animation</dfn> is as
```

Comment:
Nit: since the definition text appears to be identical to the `lt` text, do we need the `lt` attribute?

Reply:
Thanks, good catch, removed the `lt`.